### ISSUE TITLE
Unable to alter the “SetDerivedBy” property of an LayeredTileMap object in a variant entity.

### ISSUE THREAD

**Issue by mza299** [vchelaru/FlatRedBall#1414]
To be more specific - I have created a master base entity called Buildingblock in glue.

- The Buildingblock entity has a LayeredTileMap object that has the “SetByDerived” property set to true. There is no problem here.
- There are other entities that inherit from Buildingblock ; SmallHouse, BigHouse. These will be the base class for entities in the game world. For example, an entity will inherit from SmallHouse, which in turn will inherit from BuildingBlock.
- When “SetByDerived” is set to true for the entities that inherit from Buildingblock and when Glue is restarted, that property defaults back to false.

In conclusion, it seems like the “SetByDerived” property of a LayeredTileMap defaults to false of a variant entity when Glue opens/executes.

**Comment by vchelaru** [comment]
I think this is fixed, try the latest release (May 27, 2024)
